mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +27,55 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Enlarge Image
+   */
+  enlargeImage?: boolean;
```

Comment:
let's rename to  isZoomable fore more readability

Reply:
Done!